src/main.rs

```diff
@@ -37,6 +38,14 @@ fn main() -> Result<()> {
 
     for disk in &disk_parts {
         println!("{} {}", disk.parent_path.display(), disk.partition);
+
+        if options.automatically_grow {
+            println!("Growing the partition and expanding the vdev...");
+            grow::grow(&options.zpool_name, disk, options.dry_run).unwrap();
+            println!("...ok.");
+
+            println!("");
```

Comment:
Nit: Doesn't seem necessary? Or if you want a newline, just add the `\n` after the `...ok.`

Reply:
:+1: